<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="0" name="tedge_agent_user_sudo_access" skipped="1" tests="1" time="0.01" timestamp="2022-04-04T08:30:32">
	<testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments"/>
		<system-out>08:30:32 DEBUG deletedir failed to update permissions and delete /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/tedge/tedge_agent_user_sudo_access/Output/linux on this attempt: [Errno 2] No such file or directory: '/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/tedge/tedge_agent_user_sudo_access/Output/linux'
08:30:32 INFO  ==============================================================
08:30:32 INFO  Id:    tedge_agent_user_sudo_access
08:30:32 INFO  Dir:   tedge/
08:30:32 INFO  Title: Validate tedge-agent user has a limited sudo right
08:30:32 DEBUG Execution order hint: 0.0
08:30:32 INFO  ==============================================================
08:30:32 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
08:30:32 DEBUG --- test cleanup
08:30:32 DEBUG ProcessUser cleanup function done.
08:30:32 INFO  
08:30:32 INFO  Test duration: 0.01 secs
08:30:32 INFO  Test final outcome:  SKIPPED
08:30:32 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
08:30:32 INFO  
</system-out>
	</testcase>
</testsuite>
